apiary.js

```diff
@@ -1,13 +1,12 @@
-
 'use strict';
 
 (function() {
 
   // Pseudo-constants:
-  var VERSION            = '0.4.0';
+  var VERSION            = '0.5.0';
   var BASE_URL           = 'https://api.w3.org/';
   var USER_PROFILE_URL   = 'https://www.w3.org/users/';
-  var APIARY_PLACEHOLDER = /[\^\ ]apiary-([^\ ]+)/g;
+  var APIARY_PLACEHOLDER = /^apiary\-(.+)$/g;
```

Comment:
you might want to avoid special characters. Maybe use something like `/^apiray\-([\w\-]+)$/g`